lib/gap_intelligence/client/merchant_pricing_trend_downloads.rb

```diff
@@ -31,14 +31,14 @@ def merchant_pricing_trend_downloads(owner_id, params = {}, options = {}, &block
     # Deletes merchant pricing trend downloads
     #
     # @param owner_id [String,Integer] owner id of the merchant pricing trend downloads
-    # @param external_record_id [String,Integer] identity of the external record
+    # @param download_ids [Integer] id of the merchant pricing trend downloads
     # @param options [Hash] options the options to make the request with
```

Comment:
I just noticed a tiny typo, i believe it should be `# @param options [Hash] the options to make the request with`. Out of scope for this PR, but I noticed same typos in client/download and client/merchant_pricing_trend_downloads